client/src/components/feedComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import './feedComponent.css';
+import { AiFillLike, AiFillDislike, AiOutlineLike, AiOutlineDislike, AiOutlineComment } from "react-icons/ai";
+import ThumbUpIcon from '@material-ui/icons/ThumbUp';
+import ThumbDownIcon from '@material-ui/icons/ThumbDown';
+import ThumbUpOutlinedIcon from '@material-ui/icons/ThumbUpOutlined';
+import ThumbDownOutlinedIcon from '@material-ui/icons/ThumbDownOutlined';
+import InsertCommentOutlinedIcon from '@material-ui/icons/InsertCommentOutlined';
+import InsertCommentIcon from '@material-ui/icons/InsertComment';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    width: "18px",
+    padding: "4px 18px",
+    margin: "0px"
+  },
+});
+
+function Feed(props) {
```

Comment:
It would be great if you decompose the prop parameter. Not necessary but best practices.

Reply:
Also, resolve conflict.

Reply:
Do you mean by replacing **props** in the highlighted code with something like:
**{ username, image, question, answer }**?

Reply:
> Do you mean by replacing **props** in the highlighted code with something like:
> **{ username, image, question, answer }**?

Yes

Reply:
Ok, thanks!
Kindly guide me a bit in the other change you've suggested - (of adding the dummy data file)